Client/Frontend/Browser/URLBarView.swift

```diff
@@ -8,8 +8,7 @@ import Shared
 import SnapKit
 
 private struct URLBarViewUX {
-    // The color shown behind the tabs count button, and underneath the (mostly transparent) status bar.
-    static let TextFieldBorderColor = UIColor.blackColor().colorWithAlphaComponent(0.05)
+    static let TextFieldBorderColor = UIColor(rgb: 0xEEEEEE)
```

Comment:
The color should be nearly identical (a very light black on the white background == light gray). Seemed pointless to use transparency here -- I'm guessing it used to be used for something else, given the obsolete comment.